RocketChat/Rocket.Chat
# [IMPROVE] New 'not found page' design

## Proposed changes (including videos or screenshots)



- Add a new design for the not-found page
- Add English translation for "page not found" and "Homepage"
- Update English translation for "Room_not_exist_or_not_permission"
- Add "Homepage" button on the room not found page



## Issue(s)


## Steps to test or reproduce


- Log in the application
- Change the url for something that is not a RC page (e.g. https://localhost:3000/404)
- Should show the 'Not found' page with new design

## Further comments
